WalletWasabi.Gui/Controls/WalletExplorer/WalletExplorerViewModel.cs

```diff
@@ -98,16 +99,20 @@ public WalletExplorerViewModel() : base("Wallet Explorer")
 				.Where(x => x != SelectedItem)
 				.Subscribe(x =>
 				{
+					_inSelecting = true;
```

Comment:
Add try/finally here. Add `_inSelecting = false;` into the finally block.

Reply:
done